doc/changes/latest.rst

```diff
@@ -47,6 +47,8 @@ Fixes
 Enhancements
 ------------
 
+- :bdg-success:`API` The functions :func:`~image.check_niimg`, :func:`~image.check_niimg_3d` and :func:`~image.check_niimg_4d` are now part of our public API. The content of ``nilearn._utils.niimg_conversion`` and ``nilearn.image.utils`` was moved to ``nilearn.image.image``. For a smoother transition some functions from ``nilearn._utils.niimg_conversion`` will still be importable from this module till Nilearn 0.14.0 (:gh:`5788` by `Rémi Gau`_).
+
```

Comment:
```suggestion
- :bdg-success:`API` The functions :func:`~image.check_niimg`, :func:`~image.check_niimg_3d` and :func:`~image.check_niimg_4d` are now part of our public API. The content of ``nilearn._utils.niimg_conversions`` and ``nilearn.image.utils`` was moved to ``nilearn.image.image``. For a smoother transition some functions from ``nilearn._utils.niimg_conversion`` will still be importable from this module till Nilearn 0.14.0 (:gh:`5788` by `Rémi Gau`_).

```

Reply:
> * The contents of  `nilearn.image.utils` seem to remain almost the same, not moved to `nilearn.image.image`.

I thought I had moved things: I suspect I messed up something on git operation. Will do again.

> * The name of the package is changed from `nilearn._utils.niimg_conversions` to`nilearn._utils.niimg_conversion`. Is there a reason for that? Because that will do the same as moving it to `nilearn.image`, it will change the import path.

fixed

> * `For a smoother transition some functions` : Here **some** is ambiguous and gives the impression that there might be other functions still importable which is not the case.

made it very explicit which functions are still there